src/COTDStats.as

```diff
@@ -48,12 +63,18 @@ array<DivTime@> divs = { pb, div1, nextdiv, lowerbounddiv };
 
 void Render() {
 #if TMNEXT
+    if ((hideWithGame && !UI::IsGameUIVisible()) || (hideWithOP && !UI::IsOverlayShown()))
+        return;
+
     auto app = cast<CTrackMania>(GetApp());
     auto network = cast<CTrackManiaNetwork>(app.Network);
     auto server_info = cast<CTrackManiaNetworkServerInfo>(network.ServerInfo);
 
-    if (windowVisible && app.CurrentPlayground !is null && (server_info.CurGameModeStr == "TM_COTDQualifications_Online" || server_info.CurGameModeStr == "TM_TimeAttackDaily_Online")) {
-    
+    if (
+        windowVisible &&
+        app.CurrentPlayground !is null &&
+        (server_info.CurGameModeStr == "TM_COTDQualifications_Online" || (!showOnlyDuringQuali && server_info.CurGameModeStr == "TM_TimeAttackDaily_Online"))
```

Comment:
At this point I don't remember exactly, I think I just preferred to add a setting rather than change existing functionality, but I can go ahead and remove mentions of the old name along with this setting since there's no reason to show the window outside of qualifier